packages/react-ameliorate-component-button-bar/source/button-bar.js

```diff
@@ -99,7 +104,8 @@ export const ButtonBar = componentFactory('ButtonBar', ({ Parent, componentName
       return {
         ...super.resolveState.apply(this, arguments),
         ...this.getState({
-          toggledStates: {}
+          toggledStates: {},
+          lastPressedButton: null
```

Comment:
Last please change this to be an index instead
```suggestion
          activeButtonIndex: null
```